src/core/store/models/user.js

```diff
@@ -1,13 +1,27 @@
 export const user = {
   state: {
-    userBalance: 0
+    userBalance: 0,
```

Comment:
Define an object contain user info instead separate to many properties

Reply:
I wrote this code based on structure of `user` model which have merged before. If I changed it, it seems not reasonable.

Reply:
Hm ok